src/test/tools/convert_xml_to_yaml/xml2yaml.yaml

```diff
@@ -0,0 +1,40 @@
+host:
+- id: server
+  process:
+  - arguments: tgen.server.graphml.xml
+    plugin: tgen
+    starttime: 1
+- id: client
+  process:
+  - arguments: tgen.client.graphml.xml
+    plugin: tgen
+    starttime: 2
+option:
```

Comment:
Maybe call this 'options' in case we add more

Reply:
Same with 'host' and 'plugin' ?